### PR TITLE
[Fix] Scroll page position to top on navigate in settings

### PR DESCRIPTION
**Pull request by danielkryska**

### CODE REVIEW

File: s4e-web/src/app/views/settings/settings.component.ts [hunk 1]
```diff
@@ -60,6 +60,19 @@ export class SettingsComponent implements OnInit, OnDestroy {
...
       });
 
+    // Set page scroll at initial place
+    window.scrollTo(0, 0);
+    this._router.events
+      .pipe(
+        filter(event => event instanceof NavigationEnd),
+        map(() => this._router.url),
+        filter(url => url.indexOf('/settings') > -1),
+        tap(url => console.log(url)),
```
Comment (jswk) [inline s4e-web/src/app/views/settings/settings.component.ts:70]: I'd remove this log.

**Review by jswk — APPROVED** [review]
I tested it locally and it works correctly.
Apart from the log removal it is fine for merging.

File: s4e-web/src/app/views/settings/settings.component.ts (hunk 1 above)
Comment (michal-szostak) [reply s4e-web/src/app/views/settings/settings.component.ts:70]: I concur

File: s4e-web/src/app/views/settings/settings.component.ts [hunk 2]
```diff
@@ -60,6 +60,19 @@ export class SettingsComponent implements OnInit, OnDestroy {
         }
       });
 
+    // Set page scroll at initial place
+    window.scrollTo(0, 0);
+    this._router.events
+      .pipe(
+        filter(event => event instanceof NavigationEnd),
+        map(() => this._router.url),
```
Comment (michal-szostak) [inline s4e-web/src/app/views/settings/settings.component.ts:68]: I believe it's better to use url provided by `event`

**Review by michal-szostak — APPROVED** [review]
I have 2 things to improve, but generally it's good to go

**Pull request merged**